docs/source/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -111,7 +112,7 @@ To understand the cost-effectiveness of BEP and MMN supplementation for pregnant
 3.1 Causal diagram
 ------------------
 
-  .. image:: causal_dag_bep_scenarios_v2.svg
+  .. image:: causal_dag_bep_scenarios_v3_fall.svg
```

Comment:
In future efforts, you can keep the filename the same and just update the svg file... it is all in a git repository if you ever need to refer back to a previous version!